.github/workflows/validate.yml

```diff
@@ -13,6 +13,9 @@ env:
 jobs:
 
   lint:
+    permissions:
+      contents: read  # for actions/checkout to fetch code
+      pull-requests: read  # for golangci/golangci-lint-action to fetch pull requests
```

Comment:
Can be another PR, but eventually we should set permissions explicitly for all the jobs, not just `lint`?

Reply:
Yes, that can be done.